src/assets/scripts/game/GameController.js

```diff
@@ -79,6 +80,7 @@ export default class GameController {
      */
     init_pre() {
         this.game_initializeBlurFunctions();
+        this.game_initializeStartTime();
```

Comment:
I do, and it shows no issues on the file with this here. In atom, I have `linter`, `linter-eslint`, and `linter-jsonlint` for linting.